src/lib/clk.c

```diff
@@ -85,6 +85,13 @@ void clock_set_freq(int clock, uint32_t hz)
 	spin_unlock_irq(&clk_info->lock, flags);
 }
 
+void clock_low_power_mode(int clock, bool enable)
+{
+	struct clock_info *clk_info = clocks_get() + clock;
+
+	clk_info->low_power_mode(clock, enable);
```

Comment:
This should be an optional op, so it needs a check for the func ptr.

Reply:
updated